maker/src/main.rs

```diff
@@ -281,6 +281,12 @@ async fn main() -> Result<()> {
                 tracing::info!(endpoint = %http_endpoint, "HTTP interface is ready");
             })
         }))
+        .attach(AdHoc::on_request("Rocket request", |req, _data| {
+            Box::pin(async move {
+                let request = req;
```

Comment:
You could rename the closure parameter as well and save yourself this assignment.